cmd/internal/codegen/parse/parser.go

```diff
@@ -112,12 +120,11 @@ func (b *APIs) verifyRBACAnnotations() {
 				continue
 			}
 
+			// The resource name is the lower-plural of the Kind
+			resource := rs.Pluralize(strings.ToLower(gvk.Kind))
```

Comment:
`kubebuilder` now has a flag plural-kind which allows the kind to be plural. In this case, we don't need to Pluralize it again.

Reply:
We need a way to preserve this flag so that it is available outside the "create resource" command. There are couple of ways to solve this problem as discussed offline. For now, user will have to use "skip-rbac-validation" to workaround this issue.